mitosheet/mitosheet/tests/user/test_user.py

```diff
@@ -277,4 +278,25 @@ def test_upgrades_version_3_to_version_4():
 def test_main_cli_turn_off_logging():
     initialize_user()
     subprocess.run(['python', '-m', 'mitosheet', 'turnofflogging'])
-    assert get_user_field(UJ_MITOSHEET_TELEMETRY) == False
\ No newline at end of file
+    assert get_user_field(UJ_MITOSHEET_TELEMETRY) == False
+
+def test_upgrades_to_new_experiment():
+    write_fake_user_json({
+        UJ_STATIC_USER_ID: 'github_action',
+        UJ_MITOSHEET_TELEMETRY: True,
+        UJ_MITOSHEET_PRO: False,
+        UJ_EXPERIMENT: {
+            'experiment_id': 'an_old_experiment',
+            'variant': 'A',
```

Comment:
In the future, it might be nice to stick to the convention of having a variant called "experimental" and a variant called "control" just to make it really clear in our analysis what is what.

Reply:
Well... this is just a test, but for the actual test itself.

Reply:
This is something we track in the experiment tracker for now, but we can def do that in the future if we ever get into a world with more than 1 experiment or more than A/B

Reply:
I just mean instead of calling them "A" and "B", call them "experimental" and "control" so we don't have to reference the tracker to figure out which is which.